src/settings/settings.tsx

```diff
@@ -6,7 +6,6 @@
 import { createContext, ReactNode, useContext } from "react";
 import { useStorage } from "../common/use-storage";
 import { defaultCodeFontSizePt } from "../deployment/misc";
-import { stage } from "../environment";
 
 export interface Language {
```

Comment:
I'll get Emma Smart to review this (visually) and offer a view on order.

Reply:
I've reordered by English language name which Emma was happy with. Makes more sense than sorting by a technical id.